apps/src/level_base.js

```diff
@@ -1,4 +1,5 @@
 // Functions for checking required blocks.
+import {BLOCK_TYPES} from '@cdo/apps/blockly/constants';
```

Comment:
AFAIK, the "required blocks" feature a) isn't used in any new curricula and b) might not even work right any more.